ui.apps/src/main/content/jcr_root/apps/hclecomm/clientlibs/clientlib-site/less/loader.less

```diff
@@ -23,8 +24,8 @@
   position: absolute;
   left: 0;
     right: 0;
-    top: 0;
-    bottom: 0;
+    top: 40vh;
```

Comment:
@AyushHCL : why have we used vh unit here , most places its pixels right ?

Reply:
Rishab -  'px ' will fixed from the top here we want distance from the top variable so i have used vh.

Reply:
@AyushHCL : will this work on mobile devices too where height is less ?

Reply:
yes Rishab we have tested it on mobile devices also ...it will be adjusted according to view port height..